packages/web3-providers/src/alchemy/index.ts

```diff
@@ -0,0 +1,74 @@
+import { createPageable, HubOptions, NonFungibleToken, Pageable, TokenType } from '@masknet/web3-shared-base'
+import { ChainId, SchemaType } from '@masknet/web3-shared-evm'
+import urlcat from 'urlcat'
+import type { ALCHEMY_API } from '..'
+import { fetchJSON } from '../helpers'
+import { AlchemyNetworkMap } from './contants'
+import type { AlchemyNFT, AlchemyResponse } from './types'
+
+export * from './contants'
+export class AlchemyNFTAPI implements ALCHEMY_API.Provider {
+    getAssets = async (
+        address: string,
+        options: HubOptions<ChainId, Record<string, string | undefined>> | undefined,
+    ): Promise<Pageable<NonFungibleToken<ChainId, SchemaType>, Record<string, string | undefined>>> => {
+        let result: Array<NonFungibleToken<ChainId, SchemaType>> = []
+        const api_keys: Record<string, string | undefined> = {}
+        Object.keys(AlchemyNetworkMap).forEach((network) => {
+            api_keys[network] = ''
+        })
+        await Promise.all(
+            Object.keys(AlchemyNetworkMap).map(async (network, index) => {
```

Comment:
The `getAssets` method works for only one network per invocation. Please read the network chain ID from the `options`.